config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -31,7 +31,7 @@ periodics:
       - --ginkgo-parallel=25
       - --provider=gce
       - --test=false
-      - --test-cmd=/workspace/k8s.io/kubernetes/hack/run-e2e.sh
+      - --test-cmd=$GOPATH/src/k8s.io/kubernetes/hack/run-e2e.sh
```

Comment:
hummm, I'm not sure if env will be resolved here?

Reply:
you can resolve it manually by `/home/prow/go/src/k8s.io/kubernetes/hack/run-e2e.sh`...